docs/user/concepts/smoothing.rst

```diff
@@ -0,0 +1,29 @@
+
+.. _dismod-smoothing:
+
+Smoothing Continuous Functions
+------------------------------
+
+We said that rates and :ref:`covariate multipliers <covariates>` are continuous functions of age and time.
+It takes a little work to parametrize an interpolated function of age and time.
+
+ * You have to tell it where the control points are. In Cascade, we call this
+   the :py:class:`AgeTimeGrid <cascade.model.grids.AgeTimeGrid>`.
+   It's a list of ages and a list of times
+   that define a rectangular grid.
+
+ * At each of the control points of the age time grid, Dismod-AT will evaluate
+   how close the rate or covariate multiplier is to some reference value. At these
+   points, we define prior distributions. Cascade makes these *value priors*
```

Comment:
Can we link to a specific definition of value priors so the user can drill in for more information?  Perhaps: https://bradbell.github.io/dismod_at/doc/rate_table.htm#Child%20Value%20Priors.Constant and explain constant and not constant?

Reply:
OK, made a link to the priors page. And it links to the source,